cli/cmd/uninstall/uninstall_test.go

```diff
@@ -432,7 +609,8 @@ func TestUninstall(t *testing.T) {
 			messages: []string{
 				"\n==> Checking if Consul demo application can be uninstalled\n    No existing Consul demo application installation found.\n",
 				"\n==> Checking if Consul can be uninstalled\n ✓ Existing Consul installation found.\n",
-				"\n==> Consul Uninstall Summary\n    Name: consul\n    Namespace: consul\n ✓ Successfully uninstalled Consul Helm release.\n ✓ Skipping deleting PVCs, secrets, and service accounts.\n",
+				"\n==> Consul Uninstall Summary\n    Name: consul\n    Namespace: consul",
```

Comment:
is there a reason to split this?  these were intentionally grouped based on `\n==>` so that formatting was ensured.

Reply:
also effects all of the other places where this change was made.

Reply:
Great question. I split these up for tests the output now prints that the CRs were deleted. It will print that between the uninstall summary and the success messages. For example, for this test, it prints:

```

==> Checking if Consul demo application can be uninstalled
    No existing Consul demo application installation found.
    No existing Consul demo application installation found.

==> Checking if Consul can be uninstalled
  ✓ Existing Consul installation found.

==> Consul Uninstall Summary
    Name: consul
    Namespace: consul
--> Deleting custom resources managed by Consul
  ✓ Successfully uninstalled Consul Helm release.
  ✓ Skipping deleting PVCs, secrets, and service accounts.
```

Reply:
So why are taking a test that matched exact output of a section and making it less precise where output that is not visible in the test could be introduced?

Reply:
also, why is `Deleting custom resources managed by Consul` not included in this output test?  Can we change this so that:
- the test still validates the entire section in one string
- the new sentence is a part of that string

this would be more precise and follow the current pattern that was put in place because the output was inconsistent between different subcommands and mostly due to the fact that it was not under test where you could explicitly see what the expected sections and their contents would be.

Reply:
I see what you mean. I added a check for that output, thanks!